From: Adam Borowski <kilobyte@angband.pl>
Date: Sat, 23 Jul 2016 01:29:03 +0200
Subject: Add a few compiled executables to .gitignore.

Signed-off-by: Adam Borowski <kilobyte@angband.pl>
---
 .gitignore | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/.gitignore b/.gitignore
index a228908..d271efb 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,4 +1,7 @@
 *.o
 duperemove
 btrfs-extent-same
+csum-test
+hashstats
+show-shared-extents
 *~
